compte/views.py

```diff
@@ -260,3 +261,15 @@ def mes_abonnements(request):
         "compte/mes_abonnements.html",
         context={"pager": pager, "pager_base_url": "?1"},
     )
+
+
+@login_required
+def disable_reminders(request):
+    prefs = UserPreferences.objects.get(user=request.user)
+    prefs.notify_on_unpublished_erps = False
+    prefs.save()
+
+    return render(
+        request,
+        "compte/notifications_disabled.html",
+    )
```

Comment:
I know this is more work, and we only have a single preference right now, but how about having a preferences management page with a dedicated form instead? 🤔

Reply:
Yeah, I started with that, but then I checked what others do with the simple case, and I took the simplest path. But I'll do that